README.md

```diff
@@ -41,39 +41,41 @@ This tool expects to run in an environment with the following attributes:
 
 Requesting a code review:
 
-    git appraise request
+    git review request
 
 Pushing code reviews to a remote:
 
-    git appraise push [<remote>]
+    git review push [<remote>]
 
 Pulling code reviews from a remote:
 
-    git appraise pull [<remote>]
+    git review pull [<remote>]
 
 Listing open code reviews:
 
-    git appraise list
+    git review
+    git review --list
 
 Showing the status of the current review, including comments:
 
-    git appraise show
+    git review show
 
 Showing the diff of a review:
 
-    git appraise show --diff [--diff-opts "<diff-options>"] [<review-hash>]
+    git review show -p/-u/--patch [--diff-opts "<diff-options>"] [<review-hash>]
 
 Commenting on a review:
 
-    git appraise comment -m "<message>" [-f <file> [-l <line>]] [<review-hash>]
+    git review comment -m "<message>" [-f <file> [-l <line>]] [<review-hash>]
 
 Accepting the changes in a review:
 
-    git appraise accept [-m "<message>"] [<review-hash>]
+    git review accept [-m "<message>"] [<review-hash>]
 
 Submitting the current review:
 
-    git appraise submit [--merge | --rebase]
+    git review merge [--ff | --no-ff | --ff-only ]
+    git review rebase
```

Comment:
> For me the code can, and does exist without the "review" i.e. comments, analyses, CI reports.

I agree with you on this, in that yes it can exist without the review. My end goal is also to merge code. But we can't assume just because someone who has written the code and has requested a review that it is going to get merged. It might never be merged at all in some cases. I guess what I mean by this is, if you're not using git-appraise, you still have branches, you still have tags and you can still review code. For me, git-appraise creates this process where-by code is written, a review is requested, it at this point is either accepted or rejected (if rejected, with feedback), and if it is accepted it is then in a position to be submitted, at which point is is merged. And of course the best thing about it is that it is distributed. So it doesn't matter where any of the remote forks exist. I perhaps didn't explain this very well, sorry if this has been confusing.